src/code.sh

```
@@ -1,67 +1,39 @@
-if [[ ${#R1[@]} -ne ${#R2[@]} ]]; then 
-       echo "The number of R1 and R2 files for this sample are not equal - exiting"
-       exit 1
-fi
-
-# Check that each R1 has a matching R2
-# Remove "R1" and "R2" and the file suffix from all file names
-_trim_fastq_endings () {
-       # Trims the endings off R1 or R2 file names in an array, and returns as an array.
-       # Identify and cut off suffixes. Export suffix for later use.
+# set frequency of instance usage in logs to 30 seconds
+kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)